src/euphorie/client/resources/oira/help/bg/illustrations/2014-01-02-work-environment-site-related.html

```diff
@@ -13,12 +13,12 @@
 		<link
 			id="all-styles"
 			rel="stylesheet"
-			href="../../../style/all.css"
+			href="/assets/oira/style/all.css"
 			type="text/css">
```

Comment:
@thet Something has gone wrong here. Your PR will introduce a broken path in all the Help files.

Any paths that start with "/assets" need to be made relative or replaced with "++resource++euphorie.resources/". See the script https://github.com/euphorie/Euphorie/blob/main/scripts/proto2diazo.py